cow_py/contracts/abis.py

```diff
@@ -0,0 +1,860 @@
+COMPOSABLE_COW = [
```

Comment:
I'm a little confused by this? The `JSON` that is in files, can this not be used instead of declaring a var here? (or at least some method of deduplicating, ie. either define in a var, such as this and delete the JSON, or import the JSON and delete the var).

Reply:
Honestly I just thought this could be a convenient way for not needing to load the JSON files 😅  but don't think it makes much sense as we don't get the same typing benefits we'd get in TS (as far as I know at least), so I'll remove this.